.github/workflows/release.yml

```diff
@@ -0,0 +1,91 @@
+name: CI Release
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+
+jobs:
+  build:
+    name: Build and test
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ windows-latest ] 
+        dotnet: ['5.0.x']
+
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@v2
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: ${{ matrix.dotnet }}
+
+      - name: Install dependencies
+        run: dotnet restore
+      - name: Build
+        run: dotnet build --configuration Release --no-restore
+      - name: Unit test
+        run: dotnet test Amazon.IonObjectMapper.Test
+
+  sign:
+    name: Sign package
+    needs: [build]
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ] 
+        dotnet: ['5.0.x']
+      
+    steps:
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+          role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
+          role-external-id: ${{ secrets.AWS_ROLE_EXTERNAL_ID }}
+          role-duration-seconds: 900
+          
+      - name: Wait / Sleep
+        uses: jakejarvis/wait-action@v0.1.1
+      - name: Git Checkout
+        uses: actions/checkout@v2
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: ${{ matrix.dotnet }}
+        
+      - name: Create DLL
+        run: |
+          dotnet restore
+          dotnet build --configuration Release --no-restore
```

Comment:
Same here I think this would've been done by the previous build actions, no?

Also, isn't it easier to not specify dotnet restore and --no-restore?

Reply:
The previous build is ran in a separate [job](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobs) so they don't share the same environment